src/Form/AppsearchForm.php

```diff
@@ -133,8 +133,8 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       ->set('api_private_key', $form_state->getValue('api_private_key'))
       ->set('engine_name', $form_state->getValue('engine_name'))
       ->save();
-    
-      drupal_set_message($this->t('Awesome!! Connection to the endpoint is successfull using the provided key.'));
+
+    drupal_set_message($this->t('Awesome!! Connection to the endpoint is successfull using the provided key.'));
```

Comment:
`drupal_set_message()` is deprecated now. I believe ConfigFormBase already has `Drupal\Core\Messenger\MessengerTrait` use messenger trait or messenger service instead. See https://www.drupal.org/node/2774931 for more detail.

Reply:
removed